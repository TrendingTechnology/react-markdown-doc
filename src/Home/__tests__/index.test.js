import React from 'react'
import renderer from 'react-test-renderer'
import ReactTestUtils from 'react-dom/test-utils'
import { shallow } from 'enzyme'
import { Home } from '..'

import * as enzyme from 'enzyme'
import EnzymeAdapter from 'enzyme-adapter-react-16'

// Enzyme configuration
enzyme.configure({ adapter: new EnzymeAdapter() });

describe('Home Test', () => {
  test('Home component renders', () => {
    const component = renderer.create(<Home />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })


  // test('click on Button should update display', () => {
  //   const mockCallBack = jest.fn()
  //   const button = shallow((<Button />))
  //   expect(button.text()).toEqual(String(0))
  //   button.find('button').simulate('click')
  //   expect(button.text()).toEqual(String(1))
  //   button.find('button').simulate('click')
  //   expect(button.text()).toEqual(String(2))
  // })
})
