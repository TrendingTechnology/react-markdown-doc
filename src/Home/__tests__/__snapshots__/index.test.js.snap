// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Test Home component renders 1`] = `"Home"`;
